WCON_format.md

```diff
@@ -255,7 +258,7 @@ As an example, inside
       "@XJ":{ "foo": { "e": 2 }, "yes": "I think so"},
       "settings":{"q": 4, "r": 5}
     },
-    "data": { "id":1, "t":0, "x":1, "y":2, "e":3, "@XJ": {"e": 3}, "f":{"q": 4} }
+    "data": [{ "id":1, "t":0, "x":[1], "y":[2], "@XJ": {"e": 3, "f":{"q": 4}}}]
```

Comment:
My schema explicitly disallows any entries other than `"id"`, `"t"`, `"x"`, `"y"`, and anything starting with `"@"`.  So `"e"` is not just ignored, it's not allowed.  I think that's better because we already allow custom WCON entries inside the allowed `"@"` fields.

I have commented on the brackets issue in a comment at the bottom.